### ISSUE TITLE
1.4.3 Contrast (Minimum) (Loaded Data)

### ISSUE THREAD

**Issue by AlanaBayless** [UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan#72]
**Were violation(s) found?**
(If Yes, describe in the comments.)
- [ ] No violations found
- [ ] Yes, violations found and all fixed
- [ ] Yes, violations found and some fixed while others are unresolved _(Add **6.Unresolved 508** label)_
- [ ] Yes, violations found but remain unresolved _(Add **6.Unresolved 508** label)_
- [ ] Flag as a potential problem _(Add **4.Help** label)_

---
### 1.4.3 Contrast (Minimum) - Level AA
The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: 

Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;

Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Logotypes: Text that is part of a logo or brand name has no contrast requirement.

[Understanding 1.4.3](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum)

**Trusted Tester ID:**
- [ ] Test 13.C contrast

**Accessibility Principle:** Perceivable (Distinguishable)

**Comment by AlanaBayless** [comment]
![Image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/122119181/08e2404d-78f0-4d71-80e4-5c252a7a8fdf)

**Comment by AlanaBayless** [comment]
The Loaded Data page pass 1.4.3 Contrast. The 
![Image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/122119181/6c51c032-e742-4f3a-b6a9-dbfefb57e136)
